docs/How-to/Run-a-full-Terra-node/Join-public-network.md

```diff
@@ -174,3 +188,28 @@ If you plan to start a new network from the exported state, export with the `--f
 ```bash
 terrad export --height [height] --for-zero-height > [filename].json
 ```
+
+### Node sync for testing
+
+Sometimes you may want to sync faster by foregoing checks. This command should only be used by advanced users in non-production environments. To speed up the sync process during testing, use the following command:
+
+```bash
+terrad start --x-crisis-skip-assert-invariants
+```
+
+
+### Define seed nodes
+
+::: warning NOTE
+
+For more information on seeds and peers, you can [read this](https://github.com/tendermint/tendermint/blob/master/docs/tendermint-core/using-tendermint.md#peers).
+
+:::
+
+Your node needs to know how to find peers. You'll need to add healthy seed nodes to `~/.terra/config/config.toml`. The following are the current seeds for Terra mainnet:
```

Comment:
209-215 can be removed. This is covered earlier, and not needed if they use the addrbook.json

Reply:
(maybe leave line 215)